frontend/src/app/containers/App.js

```diff
@@ -1,23 +1,29 @@
-/* global ga */
+/* global ga, window */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
-import { push as routerPush } from 'react-router-redux';
-
 import cookies from 'js-cookie';
 import Clipboard from 'clipboard';
 
 
 import { getInstalled, isExperimentEnabled, isAfterCompletedDate, isInstalledLoaded } from '../reducers/addon';
+import { setState as setBrowserState } from '../actions/browser';
 import { getExperimentBySlug } from '../reducers/experiments';
 import { getChosenTest } from '../reducers/varianttests';
 import experimentSelector from '../selectors/experiment';
 import { uninstallAddon, installAddon, enableExperiment, disableExperiment, pollAddon } from '../lib/InstallManager';
+import { fetchUserCounts } from '../actions/experiments';
+import { chooseTests } from '../actions/varianttests';
 import addonActions from '../actions/addon';
 import newsletterFormActions from '../actions/newsletter-form';
 import RestartPage from '../containers/RestartPage';
+import { isFirefox, isMinFirefoxVersion, isMobile } from '../lib/utils';
+import config from '../config';
 
-const clipboard = new Clipboard('button');
+let clipboard = null;
+if (typeof document !== 'undefined') {
```

Comment:
Since this code runs at build time in node now, sometimes we don't want to do some things if we are at build time, but just want to wait until we are running in the browser.